Warning for 4.13 documentation run

## Issue Details
The documentation generation for version 4.13 with the current doxygen master gives warnings:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-4.13.1-I-141/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h:98: warning: Unsupported emoji ':FT operator()(SrcSs:' found
/home/cgal-testsuite/cgal_doc_build/CGAL-4.13.1-I-141/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h:98: warning: Unsupported emoji ':Traits:' found
```
## Source Code

In the file: `Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h` on line 98:
```
provide `TgtSs:Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an
```
this should be (like in master, note the `::` instead of `:`):
```
provide `TgtSs::Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an
```

--- a/Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h
+++ b/Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h
@@ -100,7 +100,7 @@ boost::shared_ptr<Target_skeleton> operator()( Source_skeleton const& s) const;
 \tparam SrcSs type of the source straight skeleton
 \tparam TgtSs type of the target straight skeleton
 \tparam NTConverter  a function object that must 
-provide `TgtSs:Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an 
+provide `TgtSs::Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an 
 `TgtSs::Traits::FT` which has the same value. The default value of this parameter is `NT_converter<SrcSs::Traits::FT, TgtSs::Traits::FT>`. 
 
 \cgalModels `StraightSkeletonItemsConverter_2`

